src/inc/public/AppxPackaging.hpp

```diff
@@ -1738,7 +1738,11 @@ MSIX_API HRESULT STDMETHODCALLTYPE PackPackage(
 typedef LPVOID STDMETHODCALLTYPE COTASKMEMALLOC(SIZE_T cb);
 typedef void STDMETHODCALLTYPE COTASKMEMFREE(LPVOID pv);
 
-MSIX_API HRESULT STDMETHODCALLTYPE GetLogTextUTF8(COTASKMEMALLOC* memalloc, char** logText) noexcept;
+MSIX_API HRESULT STDMETHODCALLTYPE MsixGetLogTextUTF8(COTASKMEMALLOC* memalloc, char** logText) noexcept;
+
+#ifndef GetLogTextUTF8

```

Comment:
Consider wrapping in:

#ifndef MSIX_DEFINE_GetLogTextUTF8_BACKCOMPAT
#define MSIX_DEFINE_GetLogTextUTF8_BACKCOMPAT 1
#endif

#if MSIX_DEFINE_GetLogTextUTF8_BACKCOMPAT
... current stuff ...
#endif